charts/consul/test/terraform/eks/main.tf

```diff
@@ -54,7 +54,8 @@ module "eks" {
   count = var.cluster_count
 
   source  = "terraform-aws-modules/eks/aws"
-  version = "17.20.0"
+  version = "17.24.0"
+  kubeconfig_api_version = "client.authentication.k8s.io/v1beta1"
```

Comment:
Not blocking, but was curious what does this do? I see that it's related to the error that you mentioned but was wondering why this line fixes it?